js/main.js

```diff
@@ -0,0 +1,100 @@
+function Register() {
+    this.domElement = null;
+    this.entries = $(".entries");
+    this.inputBox = $(".newEntry");
+    this.totalEl = $(".total");
+    this.total = 0.00;
+    this.form = $("form");
+    this.form.on("submit", this.updateRegister);
+
+
+    this.initialize = function(container) {
+      this.domElement = $('<div class="register"></div>');
+      container.append(this.domElement);
+  };
+
+    this.render = function() {
+
+      var templateString =
+      "<div class='ticket'>" +
+      "<h1>Thank You!</h1>" +
+      "<table><tbody class='entries'></tbody>" +
+      "<tfoot>" +
+      "<tr>" +
+      "<th>Total</th>" +
+      "<th class='total'>$0.00</th>" +
+      "</tr>" +
+      "</tfoot>" +
+      "</table>" +
+      "</div>" +
+      "<form class='entry'>" +
+      "<input class='newEntry' autofocus placeholder='How Much?''>" +
+      "</form>" +
+      "<div>";
+
+      this.domElement.html(templateString);
+
+      this.updateRegister = function() {
+        event.preventDefault();
+
+        // get the amount and clear the box
+        var amount = parseFloat(this.inputBox.val());
+        inputBox.val("");
```

Comment:
You've got the right idea, but in this case you want to use `this.domElement.find(".newEntry")` here to scope the search to just this component.